tools/cli/air-trace/README.md

```diff
@@ -0,0 +1,50 @@
+# The `air-trace` utility
+
+The main propose of this utility is to run AquaVM with data and analyze its performance.  It has two subcommands: `run` and `stats`.
+
+## `air-trace run`
+
+Executes an AIR script with data in WASM AquaVM.  It has two modes of parameter input: plain and anomaly.
+
+### Common parameters
+All common parameters are optional.  Their position is before the mode selector (`--plain` or `--anomaly`).
+
++ `--call_results PATH` parameter allows you to provide call results for current execution.
++ `--current-peer-id STR` by default is "some_id".
++ `--max-heap-size N` defines maximum heap size for WASM runtime.
++ `--runtime PATH` option defines the AquaVM WASM binary to be executed.  By default, it is "target/wasm32-wasi/release/air_interpreter_server.wasm", but you can define a global value with the `AIR_WASM_RUNTIME_PATH` environment variable.
++ with the `--json` option, tracing info is output in machine-readable JSON format.  The output can be later processed with `air-trace stats` subcommand.
++ `--tracing-params` defines tracing crate logging levels.  By default, it is equal to `info` and does trace the most high-level AquaVM constructions (data parsing, AIR script parsing, execution, result construction).  With `debug` level it traces some individual commands, and with `trace` level it traces event more fine grained functionality.
+
+The important option is `--native`.  It runs the AquaVM as the native code that can be profiled with any native profiler.  As input data deserialization and serialization time can be comparable to particle execution time, and short execution times provide less reliable results, one can use `--repeat N` option to repeat particle execution several times.  Execution result is not printed in this case, so you may run `--repeat 1` to suppress it.
```

Comment:
Less work is preformed within `--repeat`, cleaner the profiler report is.

Reply:
I dont think that storing an interpreter resullt can have a meaningful effect. We can allocate array of results first, then just move the result into vector instead of dropping it, then use the vector after repeats.

Reply:
Good idea.  However, it wasn't intended as testing utility, but performance-related.  @mikevoronov , your opinion?